Kickstarter-iOS/Locales/de.lproj/Localizable.strings

```diff
@@ -247,6 +247,7 @@
 "Manage_your_pledge_below" = "Folgenden Beitrag verwalten";
 "Manage_your_reward" = "Belohnung verwalten";
 "Message_backer" = "Nachricht an Unterstützer senden";
+"Message_banner.accessibility.Double_tap_to_dismiss" = "Double tap to dismiss.";
```

Comment:
Currently this string is not localized, but based on the conversation we had with @peat it should not delay merging stuff to `master`/:shipit: